ansible/roles/debops.tinc/templates/etc/tinc/network/tinc-up.j2

```diff
@@ -12,6 +13,14 @@ hwaddr_prefix="{{ item.value.hwaddr_prefix | d(tinc__hwaddr_prefix) }}"
 link_type="{{ item.value.link_type|d() }}"
 link_metric="{{ item.value.metric|d() }}"
 
+{% set tinc__tpl_accept_ra = '1'
+       if item.value.accept_ra|d(True) == True
```

Comment:
You cannot use Jinja directly in shell scripts, they are checked using `shellcheck` and that breaks the test. Check a bit below to see how you can wrap Jinja in a variable.

Reply:
OH! That's what that's for. Thank you!